app/assets/v2/js/user-search.js

```diff
@@ -92,8 +92,21 @@ function userSearch(elem, showAddress, theme, initialData, allowClear, suppress_
       return selected;
     }
 
-    function formatUserSelectionWithAddress(user) {
+    function formatUserSelectionWithAddress(base_user) {
       let selected;
+      let existent_properties = {};
+
+      if (base_user.element) {
+        let attr = base_user.element.attributes;
```

Comment:
`const`

Reply:
:thinking: maybe, we dont want alterate the attributes object, so const makes more sense than 'let`.